packages/katello/rubygem-pulp_rpm_client/rubygem-pulp_rpm_client.spec

```
@@ -6,7 +6,7 @@
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 3.10.0
-Release: 1%{?dist}
+Release: 0.1%{?dist}
```

Should this use `%{foremandist}`? That's what we've been using to deal with multiple Foreman releases that both tag the same version. We should then do the same in nightly and 2.5.
```suggestion
Release: 0.1%{foremandist}%{?dist}
```

---

It kinda feels odd to use foremandist in RPMs that are not related to Foreman at all. But we could.

---

It's probably worth a broader conversation, since we will probably have these ruby27 <-> ruby25 issues untill 2.4 goes away.  I wouldn't block this PR on it.

---

I think we should have the broader discussion sooner rather than later, but I won't block this PR on it.

---

agreed